### PR TITLE
update ruby codes(v0.1.5)

### PR DESCRIPTION
**Pull request by linr211**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# load `rake build/install/release tasks'
```
Comment (Xiaochao8) [inline Rakefile:1]: remove this line？

File: history.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# 0.0.1
+- Support singleton client configuration
```
Comment (Xiaochao8) [inline history.md:2]: Capitalize the first letter
of 'singleton'?

**Review by Xiaochao8 — APPROVED** [review]

File: Rakefile (hunk 1 above)
Comment (linr211) [reply Rakefile:1]: will fix it in next merge

File: history.md (hunk 2 above)
Comment (linr211) [reply history.md:2]: will fix it in next merge

**Review by jin-jasper — APPROVED** [review]

**Pull request merged**